pyanaconda/modules/payloads/payload/payload_base.py

```diff
@@ -135,7 +135,7 @@ def set_sources(self, sources):
                                    "initialized! Please tear down the sources first.")
 
         self._sources = sources
-        log.debug("New sources %s was added.", sources)
+        log.debug("New sources %s were added.", sources)
```

Comment:
It would be good to implement the `__str__` method of `PayloadSourceBase` to get some useful info here.

Reply:
Fixed.